.misc/deploy.testpypi.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Do not use `set -x` here as then it displays the PYPIPW in logs
+set -e
+
+# Get environment variables, readily decrypted by rultor
+
+
+# Ship it!
+echo "Uploading coala to test-pypi"
+pip3 install twine wheel
+python3 setup.py sdist bdist_wheel
+# Upload one by one to avoid timeout
+# Don't block merging because pypi is in error 500 mood again
+set +e
```

Comment:
this isnt good enough. It should gracefully fail only if it is a server error(500), and not if it is any error.
See https://github.com/coala/coala/issues/3644